bin/registerServerTopology.php

```diff
@@ -246,7 +246,7 @@
 
 echo $summary;
 
-$proceed = askQuestion('Do you want to register this server with those information ? (y/n)');
+$proceed = askQuestion('Do you want to register this server with those information ? (y/n) ');
```

Comment:
In English we shouldn't use space before ponctuation
```suggestion
$proceed = askQuestion('Do you want to register this server with those information? (y/n) ');
```